.github/workflows/interface-unit-tests.yml

```diff
@@ -96,20 +96,20 @@ jobs:
             cat >matrix_max_parallel.json <<-EOF
           {
             "default": 1,
-            "tf-tests": 3,
-            "core-tests": 3,
+            "core-tests": 5,
```

Comment:
Is this the number of runners per test type?

Reply:
Yes. I think core tests now take around a 100 minutes ==> split into 5.

Reply:
will have to chat with @rashidnhm, I think we're on a bit of a budget with workers per PR

Reply:
Increasing workers from ~30 to ~35 doesn't seem to bad compared with Lightning ~80 workers, but yeah let's ask @rashidnhm to weigh in.

Reply:
It's not about the total number of workers you use but how many you use *at once*. If we save a lot of time bumping this then it might be worth the trade off.


Also ... I see `tf-tests: 3` was removed? Are we okay with tf-tests only using 1 worker at a time?

Reply:
It moved down when I sorted the fields but it should still be there.

Reply:
then this sounds good to me!